### ISSUE TITLE
#issue adb device offline

### ISSUE THREAD

**Issue by RedMoonOTInetwork** [Genymobile/scrcpy#200]
Sometimes, when i use `adb devices` in the directory of scrcpy, He returns me " **12345c** (serial number of device) **offline**" and when this appears, scrcpy stops. I think you're using an old version of adb in the precompiled zip for Windows.

**Comment by rom1v** [comment]
adb version

**Comment by RedMoonOTInetwork** [comment]
Yes i Will try but i use your adb Who is in the precompiled zip with Dependencies for Windows. I Guess the adb file it's old

**Comment by RedMoonOTInetwork** [comment]
https://tlgur.com/d/q8BBp0x8

**Comment by RedMoonOTInetwork** [comment]
Version 1.0.39

**Comment by RedMoonOTInetwork** [comment]
I tried yesterday to revoke and restore to enabled USB debugging for my PC and for now it seems to work
